src/popup/components/TransactionLabel.vue

```diff
@@ -207,7 +203,7 @@ export default defineComponent({
         };
       }
 
-      const translation = transactionListTypes[txType.value!] || transactionTypes[txType.value!];
+      const translation = !t(`transaction.listType.${txType.value!}`).includes('listType') ? t(`transaction.listType.${txType.value!}`) : t(`transaction.type.${txType.value!}`);
```

Comment:
1. Please add a `// TODO` comment that will tell others to refactor it from dynamic translation keys into a map of translations
2. `t(`transaction.listType.${txType.value!}`).includes('listType')` - I don't see how it's possible to have `listType.listType`
3. Please split this line:
```
const translation = (!t(`transaction.listType.${txType.value!}`).includes('listType'))
  ? t(`transaction.listType.${txType.value!}`)
  : t(`transaction.type.${txType.value!}`);
```

Reply:
1. Willl add a TODO comment there. 
2. It's not possible to have listType.listType but if there is no translation for listType (so the text will be the fallback ex. listType.sth) it will use transaction.type translation instead of the fallback.
3. Will do

Reply:
It works fine but that could be improved together with new way to implement dynamic keys as map.